multichat/chat/consumers.py

```diff
@@ -1,6 +1,8 @@
 import json
 from channels import Channel
 from channels.auth import channel_session_user_from_http, channel_session_user
+
+from .settings import MSG_TYPE_LEAVE, MSG_TYPE_ENTER, NOTIFY_USERS_ON_ENTER_OR_LEAVE_ROOMS
```

Comment:
Like @blazaid commented (and also discussed [here](http://stackoverflow.com/a/8429282)), I also think this way is more effective to have isolated apps/modules inside a django app. That way if anyone would copy/paste the chat app to its own projects, they can declare its config in several ways.

Kind regards!